storybook/src/components/Dialog/Dialog.stories.tsx

```diff
@@ -65,9 +62,46 @@ export const Default: Story = {
   },
 }
 
+export const FormDialog: Story = {
+  args: {
+    open: true,
+    footer: (
+      <>
+        <Button type="submit" form="dialog1" value="continue">
```

Comment:
Do we need a `value` here? We don't really explain that anywhere

Reply:
Ah I should explain though, thats how form method dialog passes an optional value in `returnValue`.

Reply:
I know, but we don't say that to our users. So now it's something we set without explaining why. And do we need it in this case? The default is an empty string, which is fine here because we don't use `returnValue`

Reply:
We don't use it but if you're working on something functional you might need to know if the user clicks yes or no.

Reply:
My point is: we should either explain what it does, or not use it at all. Using it without an explanation is confusing.